pkg/util/trivy/scanner.go

```diff
@@ -0,0 +1,213 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Should we move this package to `pkg/sbom/collectors/trivy`? Also should `scanner.go` be in `pkg/sbom` directly? I agree we should keep the interface for Mocking and stuff, but there's nothing Trivy-specific in this as this part is handled by the `Collector` interface.